README.md

```diff
@@ -5,3 +5,5 @@
 This is an opinionated library to share code and approach to Datadog logging in prima.it
 
 Refer to the [official docs](https://docs.rs/prima_datadog) for help on how to setup the library in your project
+
+❕ Please note that `prima_datadog.rs` uses [DogstatsD](https://docs.datadoghq.com/developers/dogstatsd/), which means metrics will be sent using the **UDP** protocol on port `8125`. You can find more information on the official Datadog documentation
```

Comment:
Extremely petty nitpick incoming:
```suggestion
❕ Please note that `prima_datadog.rs` uses [DogstatsD](https://docs.datadoghq.com/developers/dogstatsd/), which means metrics will be sent using **UDP** on port `8125`. You can find more information on the official Datadog documentation
```

technically this is correct, but the protocol that runs on udp is the statsd protocol, and the "the" makes it sound like udp is the highest protocol in the stack? It just sounds a bit awkward to me